src/main/java/hudson/plugins/scm/koji/model/RPM.java

```diff
@@ -86,6 +92,29 @@ public boolean hasUrl() {
         return url != null;
     }
 
+    public void setHashSum(File file) throws IOException, NoSuchAlgorithmException {
```

Comment:
It's bad to just throw these exceptions. You can end with open file and in some strange state. And again, setter should not throw exceptions like this.